### ISSUE TITLE
[Feature] Manually tune font rendering options in 2.0.0

### ISSUE THREAD

**Issue by floppym** [FreeRDP/FreeRDP#6059]
**Describe the bug**
Consolas font is fuzzy in xfreerdp when connected to a Windows 10 machine with font smoothing enabled.

**To Reproduce**
1. Connect to a Windows 10 machine with font smoothing enabled. Example command line:
`xfreerdp /cert-ignore /bpp:32 /size:1400x1050 -wallpaper +fonts /clipboard /v:hostname`
2. Open an application that uses the Consolas font. An especially good example of this is the ILSpy .NET decompiler shown in the screenshots below.
3. Observe fuzzy characters.

**Expected behavior**
Characters should look nice and sharp without color bleeding.

**Screenshots**
Expected sharp rendering:
![sharp](https://user-images.githubusercontent.com/504287/79027461-4b5ed280-7b5a-11ea-98f0-6acb43bbed21.png)

Fuzzy rendering:
![fuzzy](https://user-images.githubusercontent.com/504287/79027472-56196780-7b5a-11ea-8851-8217d6bc419d.png)

**Desktop (please complete the following information):**
 - OS: Gentoo Linux

**Additional context**
I bisected this, and this was the result:

```
There are only 'skip'ped commits left to test.
The first bad commit could be any of:
b3a642164a770179aaa04315f9c2fd28d450ba3f
4766f5204bb78e0110e02ad6385d0af40c955687
61d2e84fdc03dbd90537c9c4451486bdfc51aaf9
1c19de5a76f4e32d3ece71d790c23330add8cd60
We cannot bisect more!
```

Based on these commits, I think https://github.com/FreeRDP/FreeRDP/pull/5835 is to blame. @akallabeth could you please take a look?

**Comment by akallabeth** [comment]
@floppym simple check, try with /bpp:24 then another codec is used.

**Comment by akallabeth** [comment]
the second thing (requires a custom compile):
Change the following to `FALSE`:
```
settings->DrawAllowColorSubsampling = TRUE;
settings->DrawAllowDynamicColorFidelity = TRUE;
```

This is the only relevant change from the pr you reference. It enables some features on the server that influence rendering.

**Comment by floppym** [comment]
> try with /bpp:24 then another codec is used.

That works.

> Change the following to FALSE

This also works. Here are some screen shots to compare.
I like the output with both set to false the best, but that might be because I'm most used to it.

```
    settings->DrawAllowColorSubsampling = TRUE;
    settings->DrawAllowDynamicColorFidelity = TRUE;
```
![DrawAllowColorSubsampling_DrawAllowDynamicColorFidelity](https://user-images.githubusercontent.com/504287/79045138-910db080-7bd7-11ea-8ba4-c619da06675a.png)
```
    settings->DrawAllowColorSubsampling = FALSE;
    settings->DrawAllowDynamicColorFidelity = FALSE;
```
![Both_Disabled](https://user-images.githubusercontent.com/504287/79045175-be5a5e80-7bd7-11ea-8dae-a27bbb6657cd.png)
```
    settings->DrawAllowColorSubsampling = FALSE;
    settings->DrawAllowDynamicColorFidelity = TRUE;
```
![DrawAllowDynamicColorFidelity](https://user-images.githubusercontent.com/504287/79045190-cf0ad480-7bd7-11ea-8661-ed17328576b3.png)
```
    settings->DrawAllowColorSubsampling = FALSE;
    settings->DrawAllowDynamicColorFidelity = TRUE;
```
![DrawAllowColorSubsampling](https://user-images.githubusercontent.com/504287/79045201-d7630f80-7bd7-11ea-95c2-af475975215d.png)

**Comment by akallabeth** [comment]
Ok, renamed this issue and put it on the list to allow customization of these options.

**Comment by akallabeth** [comment]
with #6063 this is configurable now.

**Comment by floppym** [comment]
Tested successfully with `/tune:FreeRDP_DrawAllowDynamicColorFidelity:false,FreeRDP_DrawAllowColorSubsampling:false`.

**Comment by akallabeth** [comment]
Fixed with #6152 and #6151
